file.go

```diff
@@ -20,7 +20,8 @@ func MarshalD3ToMemory(d3Data D3) {
 	if err != nil {
 		log.Fatalf("Unable to marshal graph data to in-memory JSON: %s", err.Error())
 	}
-	D3GraphData = strings.Replace(string(d3JSON), "null", "{}", -1)
+	//D3GraphData = strings.Replace(string(d3JSON), "null", "{}", -1)
```

Comment:
This can be eradicated, another reference is already available a few lines after (L46).